EIPS/eip-2384.md

```diff
@@ -0,0 +1,40 @@
+---
+eip: 2384
+title: Istanbul/Berlin Difficulty Bomb Delay
+author: Eric Conner (@econoar)
+discussions-to: https://gitter.im/ethereum/AllCoreDevs?at=5dd4ce8050010612b2eeb9d6
+type: Standards Track
+category: Core
+status: Draft
+created: 2019-11-20
+---
+
+## Simple Summary
+The average block times are increasing due to the difficulty bomb (also known as the "_ice age_") and slowly accelerating. This EIP proposes to delay the difficulty bomb for another 4,000,000 blocks (~611 days).
+
+## Abstract
+Starting with `???_FORK_BLKNUM` the client will calculate the difficulty based on a fake block number suggesting to the client that the difficulty bomb is adjusting 9 million blocks later than the Homestead fork, which is also 7 million blocks later than the Byzantium fork and 4 million blocks later than the Constantinople fork.
+
+## Motivation
+The difficulty bomb started to become noticable again on October 5th 2019 at block 8,600,000. Block times has been around 13.1s on average and now as of block 8,900,000 are around 14.3s. This will start to accelerate exponentially every 100,000 blocks. Estimating the added impact from the difficulty bomb on block times shows that we will see 20s block times near the end of December 2019 and 30s+ block times starting February 2020. This will start making the chain bloated and more costly to use. It's best to delay the difficulty bomb again to around the time of expected launch of the Eth2 finality gadget.
```

Comment:
Minor spelling issue according to the spellchecker: "noticable" -> "noticeable"

That's the reason the build fails.